src/client/components/results/MovieGrid.jsx

```diff
@@ -39,11 +39,13 @@ class MovieGrid extends React.Component {
                 </div>
             );
         }
-        if (typeof this.props.items === 'string') {
+        if (typeof this.props.items === 'string' || this.props.items.length === 0) {
             return (
                 <div className="section movies-gid-content mt1">
                     <div className="section-content">
-                        Noting found.
+                        <div className="message-not-found">
+                            Noting found.
```

Comment:
@vitkuz i'm being a bit nasty but it's `Nothing` :)

Reply:
Done )